### DISCUSSION TITLE
astro-parcel transformer

### DISCUSSION THREAD

**Discussion by aminya** [parcel-bundler/parcel#8272] [Ideas]
I have made a command-line package that allows building/optimizing the Astro projects using Parcel. 
https://github.com/aminya/astro-parcel

I wonder if it is possible to make this a transformer plugin

**Comment by mischnic** [comment]
When I tried to "rebuild" Eleventy with Parcel: https://github.com/mischnic/parcel-ssg, and ran into some [problems/restrictions caused by Parcel's architecture](https://github.com/mischnic/parcel-ssg#problems). I imagine Astro would be similar in that regard, it's "just" a different language the sources are authored in
